src/createServer.js

```diff
@@ -1,3 +1,62 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+const http = require('http');
+const { convertToCase } = require('./convertToCase');
+
+function createServer() {
+  return http.createServer((req, res) => {
+    const url = new URL(req.url, `http://${req.headers.host}`);
+    const originalText = url.pathname.slice(1);
+    const targetCase = url.searchParams.toCase;
```

Comment:
```suggestion
    const targetCase = url.searchParams.get('toCase');
```
You can't get toCase entry directly. You need to use [get method](https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams).